### PR TITLE
Adds scientist screams

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
I said I'll do it even if the manual screaming PR is not merged
I was this close for it to check resident science man's ckey too
Adds scientist-specific Half Life screams
Funny > Balance, I know this can be a way to check if someone sneaking around is a scientist but come on
6 kinds of screams, no I don't think I can make these play from the description you have to click them but like open them in a new tab because if you click them it will redirect THIS VERY TAB to the audio file
Male:
[ack](https://raw.githubusercontent.com/vgstation-coders/vgstation13/d0c720e24226ead44682ac8d81565aef502e04fa/sound/misc/science_scream1.ogg)
[cat screaming](https://raw.githubusercontent.com/vgstation-coders/vgstation13/d0c720e24226ead44682ac8d81565aef502e04fa/sound/misc/science_scream2.ogg)
[AaaaaAaaa](https://raw.githubusercontent.com/vgstation-coders/vgstation13/d0c720e24226ead44682ac8d81565aef502e04fa/sound/misc/science_scream3.ogg)
[AaaaaAaAaaa](https://raw.githubusercontent.com/vgstation-coders/vgstation13/d0c720e24226ead44682ac8d81565aef502e04fa/sound/misc/science_scream4.ogg)
[AaaaaaaaAaaaa](https://raw.githubusercontent.com/vgstation-coders/vgstation13/d0c720e24226ead44682ac8d81565aef502e04fa/sound/misc/science_scream5.ogg)
[aaaaaAaaaa](https://raw.githubusercontent.com/vgstation-coders/vgstation13/d0c720e24226ead44682ac8d81565aef502e04fa/sound/misc/science_scream6.ogg)
:cl:
 * rscadd: Scientists now have Half-Life screams.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Meme PR

**Review by MadmanMartian — CHANGES_REQUESTED** [review]
fix those femsci screams. Jesus christ. They're females not chipmunks.

**Comment by boy2mantwicethefam** [comment]
@MadmanMartian It's the future Deity chose

**Comment by Irradiation** [comment]
Deity isn't the be-all-end-all of design decisions. That female scream can go burn in whatever unholy hell spawned you.

**Comment by Durgen** [comment]
Do only science personnel use these?

**Comment by boy2mantwicethefam** [comment]
@Durgen Yes.
Anyway female scientists can no longer scream high-pitched variants of the normal science sounds.

File: code/modules/mob/living/carbon/emote.dm [hunk 1]
```diff
@@ -281,11 +283,15 @@
 					sound = pick(birb_sounds)
 
 				else
-					switch(H.gender)
-						if (MALE)
-							sound = pick(male_sounds)//AUUUUHHHHHHHHOOOHOOHOOHOOOOIIIIEEEEEE
-						if (FEMALE)
-							sound = pick(female_sounds)
+					if(H.mind.assigned_role in science_positions)
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/emote.dm]: April fools check, or close.

**Review by MadmanMartian — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/emote.dm (hunk 1 above)
Comment (boy2mantwicethefam) [reply code/modules/mob/living/carbon/emote.dm]: k but in a bit

File: code/modules/mob/living/carbon/emote.dm [hunk 2]
```diff
@@ -281,11 +283,15 @@
 					sound = pick(birb_sounds)
 
 				else
-					switch(H.gender)
-						if (MALE)
-							sound = pick(male_sounds)//AUUUUHHHHHHHHOOOHOOHOOHOOOOIIIIEEEEEE
-						if (FEMALE)
-							sound = pick(female_sounds)
+					if(Holiday == APRIL_FOOLS_DAY && H.mind.assigned_role in science_positions)
+						if(MALE)
```
Comment (Exxion) [inline code/modules/mob/living/carbon/emote.dm:287]: lmao

Comment (MadmanMartian) [reply code/modules/mob/living/carbon/emote.dm:287]: Code(male)

**Pull request merged**